src/components/Nuclie/Product.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Tees, HeartIcon, CartIcon } from "../../assets";
+
+const Product = (props) => {
+  const { title, desc, price, prevPrice, discount, img } = props;
+
+  return (
+    <div className='pd-card-container vertical'>
+      <div className='card-img-wrapper vertical-img'>
+        <a href='./'>
+          <img
+            src={img}
+            className='pd-img card-img vertical-card-img'
+            alt={`${img} + img`}
```

Comment:
Must change: you are writing the path in the alt of img tag instead of the name of the image.